ubuntu_desktop_installer/lib/i18n.dart

```diff
@@ -17,14 +18,11 @@ class UbuntuLocalizations {
     });
   }
 
-  static UbuntuLocalizations of(BuildContext context) {
-    return Localizations.of<UbuntuLocalizations>(context, UbuntuLocalizations);
-  }
+  static UbuntuLocalizations of(BuildContext context) =>
```

Comment:
I'm not convinced these {} to => changes are necessarily good practice. I'd prefer we kept the brackets. Please could you revert all of these.

Reply:
sure thing, I will